allauth/account/app_settings.py

```diff
@@ -315,6 +315,10 @@ def SALT(self):
     def PRESERVE_USERNAME_CASING(self):
         return self._setting("PRESERVE_USERNAME_CASING", True)
 
+    @property
+    def EMAIL_CONFIRMATION_UNVERIFIED_ONLY(self):
+        return self._setting("EMAIL_CONFIRMATION_UNVERIFIED_ONLY", True)
```

Comment:
Wondering why would this need to be a setting, as having a setting means that there are good reasons to turn this switch either way.  Is there a good reason why `True` should not be the only/default behaviour ?

Reply:
You're right. I don't think there needs to be a setting here and it should be True all the time. This setting can be removed then.